pkg/noun/jets/e/scow.c

```diff
@@ -234,159 +218,34 @@ _print_p(u3_atom cor, u3_atom p)
   return u3nc('~', list);
 }
 
-static
-u3_noun
-_print_ud(u3_atom ud)
+u3_atom
+u3qe_scow(u3_atom a, u3_atom b)
 {
-  // number of characters printed "between" periods.
-  c3_i between = 0;
-  u3_noun list = 0;
-
-  // increase input refcount to be consumed in u3ka_div(), which will free each
-  // intermediary state.
-  u3k(ud);
-
-  do {
-    if (between == 3) {
-      list = u3nc('.', list);
-      between = 0;
-    }
-
-    list = u3nc(u3ka_add(u3qa_mod(ud, 10), '0'), list);
-    between++;
-    ud = u3ka_div(ud, 10);
-  } while (ud != 0);
-
-  return list;
-}
-
-static
-u3_noun
-_print_uv(u3_atom uv)
-{
-  // number of characters printed "between" periods.
-  c3_i between = 0;
-  u3_noun list = 0;
-
-  // increase input refcount to be consumed in u3ka_div(), which will free each
-  // intermediary state.
-  u3k(uv);
-
-  do {
-    if (between == 5) {
-      list = u3nc('.', list);
-      between = 0;
-    }
-
-    c3_y tig = u3qa_mod(uv, 32);
-    list = u3nc(to_digit(tig), list);
-    between++;
-    uv = u3ka_div(uv, 32);
-  } while (uv != 0);
-
-  return u3nt('0', 'v', list);
-}
-
-static
-u3_noun
-_print_uw(u3_atom uw)
-{
-  // number of characters printed "between" periods.
-  c3_i between = 0;
-  u3_noun list = 0;
-
-  // increase input refcount to be consumed in u3ka_div(), which will free each
-  // intermediary state.
-  u3k(uw);
+  switch (a) {
+    //  XX disabled due to memory corruption
```

Comment:
can you elaborate?

Reply:
It's not a new issue, see urbit/urbit#5161. `~ponmep-litsem` is working on new implementations based on this branch.